### PR TITLE
WL-371 Update XBlock SDK for Translations

### PR DESCRIPTION
**Pull request by asadiqbal08**
@mattdrayer Currently I am adding the link as provided on JIRA for WL-371 but It seems me, It need to update to XBlock translation document. 

Please review the changes.

### CODE REVIEW

**Comment by mattdrayer** [comment]
LGTM @asadiqbal08 -- the URL is correct and the resource will be updated when https://github.com/edx/edx-documentation/pull/904 merges -- so we can merge this PR after that one goes.

@nedbat FYI

File: prototype/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}/translations/README.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This translations directory is for .po|.mo files
+
+Visit XBlock Tutorial docs on Internationalization :
+https://github.com/edx/edx-documentation/blob/master/en_us/xblock-tutorial/source/edx_platform/edx_lms.rst
```
Comment (nedbat) [inline prototype/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}/translations/README.txt]: Looks like you plan to change this to a readthedocs URL once it's ready, which is good.  I'd spell out ".po|.mo files", we don't need a cryptic notation. And while we're at it, "Visit the XBlock Tutorial docs on internationalization:"

**Comment by nedbat** [comment]
Thanks for adding this to the prototype.  :+1:

File: prototype/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}/translations/README.txt (hunk 1 above)
Comment (mattdrayer) [inline prototype/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}/translations/README.txt]: I updated the copy to be a little more clear, and changed the reference to point to the actual RTD location that will be updated by edx/edx-documentation#904

**Comment by mattdrayer** [comment]
PR edx/edx-documentation#904 has merged, and I am now going ahead with this follow-on merge

**Pull request merged**